src/group_theory/subgroup/basic.lean

```diff
@@ -2105,24 +2109,18 @@ lemma comap_ker (g : N →* P) (f : G →* N) : g.ker.comap f = (g.comp f).ker :
 @[simp, to_additive] lemma restrict_ker (f : G →* N) : (f.restrict K).ker = f.ker.subgroup_of K :=
```

Comment:
Let's not create inconsistencies between `restrict` and `range_restrict`:
```suggestion
@[simp, to_additive] lemma ker_restrict (f : G →* N) : (f.restrict K).ker = f.ker.subgroup_of K :=
```

Reply:
I don't really have a preference between `ker_` and `_ker` but it looks like `ker_` is slightly more popular throughout mathlib so it's good to go with that.